File: exam_2020_march/SuitcasesLoad.java
import java.util.Scanner;

public class SuitcasesLoad{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		
		double space = Double.parseDouble(sc.nextLine());
		
		int suitcases = 0;
		
		String input = sc.nextLine();
		
		while(!input.equals("End")){
				double suitcase = Double.parseDouble(input);
				
				if((suitcases + 1) % 3 == 0){
					
						suitcase += suitcase * 0.1;
				}
				
				if(space - suitcase < 0){
					System.out.println("No more space!");
					System.out.println("Statistic: " + suitcases + " suitcases loaded.");
					return;
				}
				space -= suitcase;
				suitcases++;
				
				if(sc.hasNext()) input = sc.nextLine();
		}
		System.out.println("Congratulations! All suitcases are loaded!");
		System.out.println("Statistic: " + suitcases + " suitcases loaded.");
	}
}

File: exam_2020_march/SuitcasesLoad.cpp
#include <iostream>

using namespace std;

int main()
{
	double space;
	cin >> space;
	cin.ignore();
	
	int suitcases = 0;
	
	string input;
	getline(cin, input);
	
	while( input != "End"){
	
		double suitcase = stod(input);
		
		if((suitcases + 1) % 3 == 0){
					
			suitcase += suitcase * 0.1;
		}
		if(space - suitcase < 0){
			cout << "No more space!" << endl;
			cout << "Statistic: " << suitcases << " suitcases loaded." << endl;
			return 0;
		}
		space -= suitcase;
		suitcases++; 
		
		getline(cin,input);
	}
	cout << "Congratulations! All suitcases are loaded!" << endl;
	cout << "Statistic: " << suitcases << " suitcases loaded." << endl;
 
	return 0;
}